build.gradle.kts

```diff
@@ -276,11 +276,11 @@ spotless {
   val ktfmtVersion: String by project
   kotlin {
     ktfmt(ktfmtVersion)
-    targetExclude("build/**")
+    target("**/*.kt")
   }
   kotlinGradle {
     ktfmt(ktfmtVersion)
-    target("*.gradle.kts", "buildSrc/*.gradle.kts")
+    target("**/*.gradle.kts")
```

Comment:
These are the only manual edits here; the rest of the changes are from ktfmt.

The `targetExclude` isn't needed because the Spotless plugin automatically excludes directories called `build` when a target path starts with `**/`, per its docs.